lib/api/core/entrypoints/embedded/index.js

```diff
@@ -97,18 +98,27 @@ class EmbeddedEntryPoint extends EntryPoint {
     this.httpServer = http.createServer();
     this.httpServer.listen(this.config.port, this.config.host);
 
-    this.protocols = {
-      http: new HttpProtocol(),
-      mqtt: new MqttProtocol(),
-      websocket: new WebSocketProtocol(),
-      socketio: new SocketIoProtocol()
-    };
+    const initPromises = [];
 
-    for (const protocol of Object.keys(this.protocols)) {
-      this.protocols[protocol].init(this);
+    for (const ProtocolClass of [
```

Comment:
Are you sure removing `this.protocols` property does not break the custom protocols implementation capability?

Reply:
`this.protocols` is not removed
What this PR does is to go back to a previous state where only active embedded protocols are stored in that object (I changed that not that long ago, when even inactive protocols where stored)

Reply:
Ok tanks for those explanations :slightly_smiling_face: